sat/data/schema/bootprep_schema.yaml

```diff
@@ -275,7 +275,7 @@ properties:
                         Filter the image or recipe to use from the given version of the given product
                         in case this product provides more than one image or recipe and one must be
                         selected.
-                      required: ['prefix']
+                      maxProperties: 1
```

Comment:
Nice, TIL. Should we also specify `minProperties: 1` to force it to be exactly one property so they can't do something like:

```
filter: {}
```

It looks like our code would handle that case just the same as if they hadn't specified `filter` at all, but maybe preventing it in the schema would be smart to reduce the likelihood of errors?

Reply:
Seems reasonable, I've added this to the schema.